testnets/shentutestnet/chain.json

```diff
@@ -0,0 +1,230 @@
+          "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.2/shentud_2.7.2_win_x86_64.exe"
+        },
+        "next_version_name": "v2.8.0",
+        "ibc": {
+          "type": "go",
+          "version": "v4.2.1",
+          "ics_enabled": [
+            "ics20-1"
+          ]
+        }
+      },
+      {
+        "name": "v2.8.0",
+        "height": 15094300,
+        "proposal": 23,
+        "tag": "v2.8.0",
+        "ibc_go_version": "v4.2.1",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "recommended_version": "v2.8.0",
+        "compatible_versions": [
+          "v2.8.0"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_linux_amd64",
+          "darwin/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_arm64_macos",
+          "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_win_x86_64.exe"
+        },
+        "next_version_name": "",
+        "ibc": {
+          "type": "go",
+          "version": "v4.2.1",
+          "ics_enabled": [
+            "ics20-1"
+          ]
+        }
+      }
+    ],
+    "ibc": {
+      "type": "go",
+      "version": "v4.2.1",
+      "ics_enabled": [
+        "ics20-1"
+      ]
+    }
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.svg"
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "8575cc152285b3eed0fd22807c2de0c10867b7f5",
+        "address": "47.252.117.255:26656",
+        "provider": "Shentu"
+      },
+      {
+        "id": "877c98831847677a340e47a0108fafc733658ee8",
+        "address": "47.253.54.177:26656",
+        "provider": "Shentu"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "2120f1d2e13a3093a84fb4c007e8391247ecff52",
+        "address": "47.253.90.228:26656",
+        "provider": "Shentu"
+      },
+      {
+        "id": "751de0c3ef07652ae6e2b5a90469439584844ca2",
+        "address": "47.253.51.210:26656",
+        "provider": "Shentu"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://yulei-rpc.shentu.org:443",
+        "provider": "Shentu"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://yulei-rest.shentu.org/",
+        "provider": "Shentu"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "yulei-grpc.shentu.org:443",
+        "provider": "Shentu"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "Shentu Explorer",
+      "url": "https://explorer.shentu.org/?chain=yulei-2.1",
+      "tx_page": "https://explorer.shentu.org/transactions/${txHash}?chain=yulei-2.1"
+    }
+  ],
+  "images": [
+    {
```

Comment:
Please use image sync, pointing to the asset. Something ike:
`"image_sync": {
  "chain_name": "shentutestnet",
  "base_denom": "uctk"
},`

Reply:
added the image_sync field